### PR TITLE
Remove Sass-related code.

### PR DESCRIPTION
**Pull request by mcbridem**
This is boilerplate code that does not belong in this repository.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,20 +1,6 @@
 var gulp = require('gulp'),
   nodemon = require('gulp-nodemon'),
-  plumber = require('gulp-plumber'),
-  livereload = require('gulp-livereload'),
```
Comment (joeyquarters) [inline gulpfile.js]: You should probably remove `gulp-livereload` as a dependency as well.

**Comment by joeyquarters** [comment]
If you use `npm uninstall gulp-plumber gulp-livereload gulp-livereload`, the `package-lock.json` should be updated correctly (assuming you are using the version of Node specified in the `engines` key of the `package.json`).

File: gulpfile.js (hunk 1 above)
Comment (mcbridem) [reply gulpfile.js]: I wasn't totally sure whether <https://github.com/theatlantic/arion/blob/e6da4743d760ab2200de2963c913ce681351e7f1/gulpfile.js#L6> and friends meant that we should continue to install this package.

**Comment by mcbridem** [comment]
> If you use `npm uninstall gulp-plumber gulp-livereload gulp-livereload`, the `package-lock.json` should be updated correctly (assuming you are using the version of Node specified in the `engines` key of the `package.json`).

I'm presuming there's a typo in this command and one of the references to `gulp-livereload` should be `gulp-sass`. I have Node 9.11.1 installed locally and we're running Node 10.14.2 on various production systems around here as of a few weeks ago; I'm not sure I'm the best person to regenerate `package-lock.json`. If anyone wants to step in here, that would be really welcome.

**Pull request merged**